### DISCUSSION TITLE
Local/template json data not refreshed when running 11ty in --serve mode (2.0.0-canary.23)

### DISCUSSION THREAD

**Discussion by javanigus** [11ty/eleventy#2720] [Q&A]
I am running 2.0.0-canary.23.

I have the following test files:
test4/test4.json
test4/test4.njk (simply displays the json data)

When I run 11ty in --serve mode, I can see the json data at test4/index.html. But if I edit the json file and reload the page, I don't see the updated json data displayed in the page. I can see that 11ty has detected a change to the test4.json and writes the test4/index.html file. I have to quit and rerun 11ty to see the updated json data. Is 11ty dev server caching the local JSON data? Is there a way to edit local JSON data and have 11ty dev server live-reload the changes?

**Comment by Gr0t3Man** [comment]
Hi,
I have countered the same problem with json files in the _data directory in canary.21 and up (tested it till canary.26). 

And reported this unexpected behavior as a bug. [2735](https://github.com/11ty/eleventy/issues/2735)
